dags/multipod/configs/maxtext_sweep_gce_config.py

```diff
@@ -89,9 +100,21 @@ def get_maxtext_sweep_gce_config(
         task_owner=test_owner,
         num_slices=curr_num_slices,
     )
+
+    job_metric_config = metric_config.MetricConfig(
+        tensorboard_summary=metric_config.SummaryConfig(
+            file_location=tb_file_location,
+            aggregation_strategy=metric_config.AggregationStrategy.MEDIAN,
+            use_regex_file_location=True,
```

Comment:
Why can't we just consume the `tb_file_location` here? Is it the resulting `tf.events` file having a random name?

Reply:
Right, the `tf.events` file has a random name and we only know the bucket and directory name.